post_installation_configuration/index.adoc

```diff
@@ -46,44 +41,37 @@ As a cluster administrator, you can modify the configuration resources of the ma
 ** Machine set creation to handle the workloads
 ** Cloud provider credential management
 
-* xref:../post_installation_configuration/configuring-private-cluster.adoc#configuring-private-cluster[Configure cluster components to be private]:
-By default, the installation program provisions {product-title} by using a publicly accessible DNS and endpoints. If you want your cluster to be accessible only from within an internal network, configure the following components to be private:
+* xref:../post_installation_configuration/configuring-private-cluster.adoc#configuring-private-cluster[Configure cluster components to be private]: By default, the installation program provisions {product-title} by using a publicly accessible DNS and endpoints. To make your cluster accessible only from within an internal network, configure the following components to be private:
 
 ** DNS
 ** Ingress Controller
 ** API server
 
-* xref:../post_installation_configuration/node-tasks.adoc#post-install-node-tasks[Perform node operations]:
-By default, {product-title} uses {op-system-first} compute machines.
-As a cluster administrator, you can perform the following operations with the machines in your {product-title} cluster:
+* xref:../post_installation_configuration/node-tasks.adoc#post-install-node-tasks[Perform node operations]: By default, {product-title} uses {op-system-first} compute machines. You can perform the following node operations:
 
 ** Add and remove compute machines
```

Comment:
These are independent sentences so a full stop could be added to each item here.